packages/beyonic/package.json

```diff
@@ -31,7 +31,7 @@
     "configuration-schema.json"
   ],
   "dependencies": {
-    "@openfn/language-common": "^1.15.1",
+    "@openfn/language-common": "^2.0.0",
```

Comment:
Why not `"@openfn/language-common": "workspace:*",`?

Reply:
I think this is just what pnpm does - but you're absolutely right!

There are a few like this, I'll update them all.